src/app/page.tsx

```
@@ -32,16 +32,23 @@ interface IWaifuResponse {
+    "https://api.animemoe.us/waifu/"
+  );
   const [hasMore, setHasMore] = useState(true);
 
   const queryClient = useQueryClient();
 
   // Initial data fetch
-  const { isLoading: isLoadingWaifuResponse, isError: isErrorWaifuResponse, error: errorWaifuResponse } = useQuery<IWaifuResponse>({
+  const {
+    isLoading: isLoadingWaifuResponse,
+    isError: isErrorWaifuResponse,
+    error: errorWaifuResponse,
```

ESLint

[error] 'errorWaifuResponse' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)